dev/extra-ml-requirements.txt

```diff
@@ -16,6 +16,8 @@ fastai>=2.4.1
 spacy
 # Required by mlflow.tensorflow
 tensorflow
+# Avoid using tensorflow-estimator 2.7 due to https://github.com/tensorflow/tensorflow/issues/52883
+tensorflow-estimator!=2.7.0
```

Comment:
https://github.com/tensorflow/tensorflow/issues/52883#issuecomment-955802164 says:

> Looks the problem is in wrong dependency in tensorflow 2.6.0. It has tensorflow-estimator~=2.6, while it shall be tensorflow-estimator~=2.6.0.
Similar thing will happen to keras, when it is not in rc phase anymore, as the same dependency (~=2.6) is set there